### DISCUSSION TITLE
unwanted warning

### DISCUSSION THREAD

**Discussion by nethunter518** [termux/termux-packages#7650] [Q&A] [answered]
I am alredy update my termux but this is display this message

    You are likely using a very old version of Termux,
probably installed from the Google Play Store.
There are plans in the near future to remove the
Termux apps from the Play Store so that new users
cannot install them and to **disable** them for
existing users with app updates to prevent the use
of outdated app versions. Instead, you are
encouraged to move to F-Droid or Github sources
(see [1]). You can backup all your current Termux
data before uninstallation and then restore it later
by following instructions in the wiki [2]. Check
the changelog [3] for all the new features and fixes
that you are currently missing. Check [4] for why
this is being done.

[1] https://github.com/termux/termux-app#installation
[2] https://wiki.termux.com/wiki/Backing_up_Termux
[3] https://github.com/termux/termux-app/releases
[4] https://github.com/termux/termux-app#google-play-store-deprecated

**Comment by agnostic-apollo** [comment, marked as answer]
The currently latest version of `termux-app` is `v0.117`. Go and read the links provided.
